src/components/Editor.jsx

```diff
@@ -72,6 +72,12 @@ class Editor extends React.Component {
     } = this.state
 
     const codeMirrorInst = this.codeMirror.getCodeMirror();
+
+    // If this is the first time through, value may be set without Codemirror
+    // knowing about it. Push the value to codemirror to rectify
+    if (codeMirrorInst.getValue() === '' && value !== '') {
+      codeMirrorInst.setValue(value);
+    }
```

Comment:
Components mount from children up, so the `value` prop is not yet set when the Editor mounts, as the SnippetArea has not yet mounted and loaded its data.